### PR TITLE
[ios] Night style.

### PR DESCRIPTION
**Pull request by VladiMihaylenko**

### CODE REVIEW

**Comment by VladiMihaylenko** [comment]
@kshalnev @igrechuhin ptal

File: iphone/Maps/Categories/UIKitCategories.mm [hunk 1]
```diff
@@ -205,6 +203,147 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event {}
...
+}
+
+@end
+
+@implementation UITableViewCell (Refresh)
+
+- (void)refresh
+{
+  [super refresh];
+  UIColor * opposite = self.selectedBackgroundView.backgroundColor.opposite;
```
Comment (igrechuhin) [inline iphone/Maps/Categories/UIKitCategories.mm]: [self.selectedBackgroundView refresh];

File: iphone/Maps/Categories/UIKitCategories.mm [hunk 2]
```diff
@@ -205,6 +203,147 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event {}
...
+    self.selectedBackgroundView.backgroundColor = opposite;
+}
+
+@end
+
+@implementation UINavigationBar (Refresh)
+
+- (void)refresh
+{
+  UIColor * oppositeTint = self.tintColor.opposite;
```
Comment (igrechuhin) [inline iphone/Maps/Categories/UIKitCategories.mm]: [super refresh];

File: iphone/Maps/Categories/UIKitCategories.mm [hunk 3]
```diff
@@ -205,6 +203,147 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event {}
...
+    self.textColor = oppositeText;
+}
+
+@end
+
+@implementation UISlider (Refresh)
+
+- (void)refresh
+{
+  UIColor * opposite = self.minimumTrackTintColor.opposite;
```
Comment (igrechuhin) [inline iphone/Maps/Categories/UIKitCategories.mm]: [super refresh];

File: iphone/Maps/Categories/UIKitCategories.mm [hunk 4]
```diff
@@ -205,6 +203,147 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event {}
...
+    self.minimumTrackTintColor = opposite;
+}
+
+@end
+
+@implementation UISwitch (Refresh)
+
+- (void)refresh
+{
+  UIColor * opposite = self.onTintColor.opposite;
```
Comment (igrechuhin) [inline iphone/Maps/Categories/UIKitCategories.mm]: [super refresh];

File: iphone/Maps/Categories/UIKitCategories.mm [hunk 5]
```diff
@@ -205,6 +203,147 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event {}
...
+@end
+
+@implementation UITextView (Refresh)
+
+- (void)refresh
+{
+  UIColor * oppositeBackground = self.backgroundColor.opposite;
+  UIColor * oppositeText = self.textColor.opposite;
+  UIColor * oppositeTint = self.tintColor.opposite;
+  if (oppositeBackground)
```
Comment (igrechuhin) [inline iphone/Maps/Categories/UIKitCategories.mm]: [super refresh];

File: iphone/Maps/Categories/UIKitCategories.mm [hunk 6]
```diff
@@ -205,6 +203,147 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event {}
...
+
+- (void)refresh
+{
+  UIColor * oppositeText = self.textColor.opposite;
+  UIColor * oppositeBackground = self.backgroundColor.opposite;
+  UILabel * placeholder = [self valueForKey:@"_placeholderLabel"];
+  UIColor * oppositePlaceholder = placeholder.textColor.opposite;
+  if (oppositeText)
+    self.textColor = oppositeText;
+  if (oppositeBackground)
```
Comment (igrechuhin) [inline iphone/Maps/Categories/UIKitCategories.mm]: [super refresh];

**Comment by igrechuhin** [comment]
@vng PTAL

**Comment by igrechuhin** [comment]
LGTM

**Comment by kshalnev** [comment]
LGTM

**Pull request merged**